index.bs

```diff
@@ -871,7 +872,15 @@ dictionary MLComputeResult {
 interface MLContext {
   Promise<MLComputeResult> compute(
```

Comment:
Maybe introduce a comment here that `compute()` is going away. There's a note later, but some of us (maybe just me?) fixate on the IDL.